### PR TITLE
Add clothing store

### PR DESCRIPTION
**Pull request by colindamelio**
Please review my code :)
Be thoughtful, constructive, and positive.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,84 +1,110 @@
+const storeApp = {};
```
Comment (melpeto) [inline scripts.js:1]: Good use of a namespace :)

File: scripts.js [hunk 2]
```diff
@@ -1,84 +1,110 @@
...
-    }, 
-    {
-      title: 'HackerYou Hoodie',
-      price: 50.00,
-      stock: 4,
-    }, 
-  ]
+storeApp.totalInventory = [
+  {
+    title: 'Bowie Tee',
```
Comment (caroforever) [inline scripts.js:5]: can we rename the bowie tee to something else?

**Review by caroforever — CHANGES_REQUESTED** [review]

File: scripts.js [hunk 3]
```diff
@@ -1,84 +1,110 @@
...
+  },
+  {
+    title: 'Don\'t Know Tee',
+    url: 'images/dontevenknow.jpg',
+    price: 22.50,
+    stock: 8,
+  },
+  {
+    title: 'Doughnut Jean Jacket',
+    url: 'images/doughnut.jpg',
```
Comment (LaurynFerkul) [inline scripts.js:18]: Great job with everything! Why don't we try using this image instead?
```suggestion
    url: 'images/Sprinkledoughnut.jpg',
```

**Review by LaurynFerkul — CHANGES_REQUESTED** [review]

**Review by melpeto — APPROVED** [review]

File: scripts.js [hunk 4]
```diff
@@ -1,84 +1,110 @@
...
+    document.querySelector('.inventory').append(listItem);
+  });
+}
 
-  // STEP THREE: display items on the page passing in USD as defualt
 
+storeApp.init = function () {
+  const buttons = document.querySelectorAll('button');
+  buttons.forEach((button) => {
+    button.addEventListener('click', () => {
```
Comment (melpeto) [inline scripts.js:95]:
```suggestion
    button.addEventListener('click', function() {
```

Comment (melpeto) [reply scripts.js:95]: The arrow function may cause problems when using the 'this' keyword. Changed to anonymous function

**Review by melpeto — CHANGES_REQUESTED** [review]

**Pull request closed without merging**